sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -6483,3 +6483,11 @@ def test_issue_15560():
     e = pretty(a*(KroneckerProduct(a, a)))
     result = 'a*(a x a)'
     assert e == result
+
+def test_issue_15583():
+    from  sympy.physics import mechanics
```

Comment:
This should go at the top of the file with all imports.

Reply:
ok ,i will do this.